src/Shared/Model/Mutators/SeparatorMutator.cs

```diff
@@ -0,0 +1,58 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// --------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.CST.OpenSource.Model.Mutators
+{
+    /// <summary>
+    /// Generates mutations of added or removed separators.
+    /// Separators are '.', '-', '_'.
```

Comment:
Maybe clarify here and say that these are the default separators, but can be modified.